etrago/appl.py

```diff
@@ -29,6 +29,7 @@
 import os
 import os.path
 import numpy as np
+from tools.utilities import drop_sectors
```

Comment:
This imported function is not used in the file currently, so I'm not sure if you really want to merge this change.